crates/engine/tree/src/tree/root.rs

```diff
@@ -714,31 +735,6 @@ fn get_proof_targets(
     targets
 }
 
-/// Calculate multiproof for the targets.
-#[inline]
-fn calculate_multiproof<Factory>(
```

Comment:
nit: would prefer to have it in a separate function still, so that it's easier to swap parallel/non-parallel proofs in the future if needed

Reply:
sure, recovered